eosclubhouse/clubhouse.py

```diff
@@ -469,6 +470,10 @@ def _close_message(self, _button):
     def _animate(self, direction):
         display = self.get_screen().get_display()
         primary_monitor = display.get_primary_monitor()
+
+        if not primary_monitor:
+            primary_monitor = display.get_monitor_at_window(self.get_window())
```

Comment:
:+1: . The code duplication is growing. I think is a good time to refactor the logic to obtain the monitor in a separate private method.